runner/package.json

```diff
@@ -94,6 +94,7 @@
     "@babel/eslint-plugin": "^7.11.3",
     "@babel/plugin-proposal-class-properties": "7.10.1",
     "@babel/plugin-proposal-export-default-from": "^7.12.1",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.13.8",
```

Comment:
Whats this plugin for?

Reply:
I'm using this in model and designer, I was going to use it for the fix but never needed to.
it allows you to do `&&=` `??=` `||=` which is new in the ECMAScript (2021) spec